app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json

```diff
@@ -6,7 +6,7 @@
     "description": "For Search",
     "version": "V0.1"
   },
-  "host": "localhost:5050",
+  "host": "localhost:64834",
```

Comment:
do not know if this will work on openshift.

Reply:
this is the spec to generate the class libraries.. actual configuration are in openshift and this is not in use.

Reply:
ok.